lib/network_utils.ml

```diff
@@ -546,6 +547,17 @@ module Dhclient = struct
 		end
 end
 
+module Dcbtool = struct
+	let call ?(log=false) args =
+		call_script ~log_successful_output:log dcbtool args
+
+	let is_fcoe_supported name =
+		let output = call ["gc"; name; "dcb"] in
```

Comment:
I don't think we can assume that `dcbtool` always present, so we need to add some error handling here.